.github/workflows/build_and_test.yml

```diff
@@ -324,62 +324,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         fuzzer:
-          - ./fuzzers/fuzzbench_fork_qemu
```

Comment:
Don't think we should remove them?

Reply:
BTW, can we have a script that checks these .yml entries map to the folders in ./fuzzers?

Reply:
Gah, this is why I rebase not merge.

Okay, let me fix.